.codacy.yml

```diff
@@ -7,7 +7,8 @@ exclude_paths:
   - 'docs/**'
   - '**/docs/**'
   - '**/pkg/proto/**'
-  - 'konnectd/assets/identifier/**'
+  - 'konnectd/ui_config/**/*'
+  - 'konnectd/scripts/**/*'
```

Comment:
remove the last `/*` to include also the files in the `ui_config` folder.
Your glob patttern only selects files inside of subfolders of `ui_config`.

See https://globster.xyz/?q=%2Fmyapp%2Fsrc%2F**&f=%2Fmyapp%2Freadme.md%2C%2Fmyapp%2Fconfig%2Fstaging.js%2C%2Fmyapp%2Fconfig%2Fproduction.js%2C%2Fmyapp%2Fsrc%2Fservices%2Futils.js%2C%2Fmyapp%2Fsrc%2Fservices%2Ftimezone.ts%2C%2Fmyapp%2Fsrc%2Fcontrollers%2Fhealth.js%2C%2Fmyapp%2Fsrc%2Fcontrollers%2Fuser.module.ts%2C%2Fmyapp%2Fsrc%2FREADME.md%2C%2Fmyapp%2Fassets%2Flogo.png%2C%2Fmyapp%2Fassets%2Flogo_small.png%2C%2Fmyapp%2Fassets%2Ffavicon.ico